march_hardware/src/PDOmap.cpp

```diff
@@ -8,125 +8,142 @@ namespace march4cpp
 {
 PDOmap::PDOmap()
 {
-  this->initAllObjects();
+  this->all_objects[IMCObjectName::StatusWord] = IMCObject(0x6041, 16);
+  this->all_objects[IMCObjectName::ActualPosition] = IMCObject(0x6064, 32);
+  this->all_objects[IMCObjectName::MotionErrorRegister] = IMCObject(0x2000, 16);
+  this->all_objects[IMCObjectName::DetailedErrorRegister] = IMCObject(0x2002, 16);
+  this->all_objects[IMCObjectName::DCLinkVoltage] = IMCObject(0x2055, 16);
+  this->all_objects[IMCObjectName::DriveTemperature] = IMCObject(0x2058, 16);
+  this->all_objects[IMCObjectName::ActualTorque] = IMCObject(0x6077, 16);
+  this->all_objects[IMCObjectName::CurrentLimit] = IMCObject(0x207F, 16);
+  this->all_objects[IMCObjectName::MotorPosition] = IMCObject(0x2088, 32);
+  this->all_objects[IMCObjectName::ControlWord] = IMCObject(0x6040, 16);
+  this->all_objects[IMCObjectName::TargetPosition] = IMCObject(0x607A, 32);
+  this->all_objects[IMCObjectName::TargetTorque] = IMCObject(0x6071, 16);
+  this->all_objects[IMCObjectName::QuickStopDeceleration] = IMCObject(0x6085, 32);
+  this->all_objects[IMCObjectName::QuickStopOption] = IMCObject(0x605A, 16);
 }
 
-void PDOmap::addObject(IMCObjectName objectname)
+void PDOmap::addObject(IMCObjectName object_name)
 {
-  if (this->allObjects.count(objectname) != 1)
```

Comment:
I think this will be easier to read if you replace it with the `find` function from `map`. This will also stop when it has found the element and not continue to count, while there can only exist one in a unique map.
```c++
if (this->all_objects.find(object_name) == this->all_objects.end())
```